sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb

```diff
@@ -0,0 +1,343 @@
+   "metadata": {},
+   "source": [
+    "## Make predictions using the endpoint\n",
+    "\n",
+    "The endpoint is now up and running, and ready to handle inference requests. The `deploy` call above returned a `predictor` object. The `predict` method of this object handles sending requests to the endpoint. It also automatically handles JSON serialization of our input arguments, and JSON deserialization of the prediction results.\n",
+    "\n",
+    "We'll use these sample images.\n",
+    "\n",
+    "| &nbsp; | &nbsp; |\n",
+    "| - | - |\n",
+    "| ![](kitten.jpg) | ![](bee.jpg) |\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# read the image files into a tensor (numpy array)\n",
+    "kitten_image = sample_utils.image_file_to_tensor('kitten.jpg')\n",
+    "\n",
+    "# get a prediction from the endpoint\n",
+    "# the image input is automatically converted to a JSON request.\n",
+    "# the JSON response from the endpoint is returned as a python dict\n",
+    "result = predictor.predict(kitten_image)\n",
+    "\n",
+    "# show the raw result\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add class labels and show formatted results\n",
+    "\n",
+    "The `sample_utils` module includes functions that can add Imagenet class labels to our results and print formatted output. Let's use them to get a better sense of how well our model worked on the input image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# add class labels to the predicted result\n",
+    "sample_utils.add_imagenet_labels(result)\n",
+    "\n",
+    "# show the probabilities and labels for the top predictions\n",
+    "sample_utils.print_probabilities_and_labels(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optional: make predictions using the second model\n",
+    "\n",
+    "If you added the second model (`mobilenet_v2_035_224`) in the previous optional step, then you can also send prediction requests to that model. To do that, we'll need to create a new `predictor` object."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.tensorflow.serving import Predictor\n",
+    "\n",
+    "# use values from the default predictor to set up the new one\n",
+    "predictor2 = Predictor(predictor.endpoint, model_name='mobilenet_v2_035_224')\n",
+    "\n",
+    "# note: if you are using \"local mode\" (a \"local\" or \"local_gpu\" instance_type), \n",
+    "# you need to create the new Predictor this way instead:\n",
+    "# predictor2 = Predictor(predictor.endpoint, model_name='mobilenet_v2_035_224', \n",
+    "#                        sagemaker_session=predictor.sagemaker_session)\n",
```

Comment:
I think this could be easier to include in the markdown above rather than in commented-out code

Reply:
ok